src/lib/crypto/kimchi_bindings/wasm/README.md

```diff
@@ -0,0 +1,29 @@
+# Kimchi WASM
+
+This code allows us to compile parts of Kimchi into [Web Assembly (WASM)](https://webassembly.org/).
+
+## Requirements
+
+For this to work, you will need to install the following dependencies:
+
+* [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/)
+* [wasm-bindgen-cli](https://rustwasm.github.io/docs/wasm-bindgen/reference/cli.html)
+
+## Usage
+
+To build for nodejs:
+
+```console
+$ wasm-pack build --mode no-install --target nodejs --out-dir ./nodejs ./. -- --features nodejs
```

Comment:
should we point to the `build.sh` scripts here? they add important stuff, like toggling wasm features, increasing the wasm memory, etc